### ISSUE TITLE
Create react app build doesn't work right with this package

### ISSUE THREAD

**Issue by paidit-app** [SheetJS/js-xlsx#1184]
When building our create-react-app generated application, we suddenly started receiving errors about a missing build file, like: `Refused to execute script <FILE> because its MIME type ('text/html') is not executable...` or `GET <FILE> 404 (Not Found)`. Upon looking at the generated **service-worker**, we noticed that the bundles's 'js/main' is indeed missing. We couldn't solve the issue in any way until we traced back the commit that started this weird behaviour. It was when we incorporated **js-xlsx** in our project, following the react instructions. When we removed the package completely, the build was successful once again.
We had no other choice, but deduce that this package is somehow changing the **react-scripts build** functionality. Our project is not ejected and we would like to keep it that way, so touching the webpack.config is something we would like to avoid. Does anybody know why is it happening and how to fix it?

**Comment by SheetJSDev** [comment]
Admittedly not too familiar with CRA -- can you share a small project or a set of steps for reproduction?

**Comment by paidit-app** [comment]
After further digging and failing to reproduce the issue, we finally managed to figure out the problem and **js-xlsx** is not at fault here. Apparently, the webpack configuration of CRA project limits the bundle size and omits the _main.js_ build file, when it grows over the limit. When inspecting the bundle, we noticed that js-xlsx takes quite a lot of space (size-wise. Was at 300+ KB), which is why upon removing it, our bundle made the cut and everything worked as expected.
Why this package weighs so much is another question, but it certainly doesn't change any functionality. Hopefully others will find this info useful.
Thanks @SheetJSDev for replying. We are closing the issue as the real problem either lies within the size of js-xlsx or the size limit of CRA configurations
